docs/src/api.md

```diff
@@ -156,18 +172,27 @@ accessor functions.
 
 ```@docs
 species
+Catalyst.get_species
 nonspecies
 reactions
+Catalyst.get_rxs
 nonreactions
 numspecies
 numparams
 numreactions
 speciesmap
 paramsmap
-isspecies
 isautonomous
-Catalyst.isconstant
-Catalyst.isbc
+```
+
+## Coupled reaction/equation system properties
+The following system property accessor functions are primarily relevant to reaction system [coupled 
+to differential and/or algebraic equations](@ref constraint_equations).
+```@docs
+ModelingToolkit.has_alg_equations
+ModelingToolkit.alg_equations
+ModelingToolkit.has_diff_equations
+ModelingToolkit.diff_equations
 ```
 
 ## Basic reaction properties
```

Comment:
The ordering is a bit awkward now. I'd put the species properties before the reaction properties and metadata.

Reply:
done